examples/chat_with_ai/main.py

```diff
@@ -5,10 +5,13 @@
 from langchain.chat_models import ChatOpenAI
 
 from nicegui import Client, ui
+from log_callback_handler import NiceGuiLogElementCallbackHandler
 
-OPENAI_API_KEY = 'not-set'  # TODO: set your OpenAI API key here
+OPENAI_API_KEY = "not-set"  # TODO: set your OpenAI API key here
 
-llm = ConversationChain(llm=ChatOpenAI(model_name='gpt-3.5-turbo', openai_api_key=OPENAI_API_KEY))
+llm = ConversationChain(
+    llm=ChatOpenAI(model_name="gpt-3.5-turbo", openai_api_key=OPENAI_API_KEY),
```

Comment:
Please do not reformat existing code if it does not change. We use a line length of 120.